python/tvm/script/parser.py

```diff
@@ -1047,6 +1056,91 @@ def transform_TypeConstant(self, node):
         """
         return node.value
 
+    def transform_TypeTuple(self, node):
+        return node.values
+
+    def transform_TypeCall(self, node):
+        """Call value visitor for TypeCall.
+
+        This method is for syntax sugar of T.match_buffer()
+        """
+
+        def parse_typecall_params(func, params, keyword_params):
+            args = []
+            for arg in params:
+                if isinstance(arg, ast.TypeTuple):
+                    values = []
+                    for value in self.transform(arg):
+                        values.append(self.transform(value))
```

Comment:
would be better to recursively transform elements of `TypeTuple` in `transform_TypeTuple` than do it here, you can find examples in `transform_Tuple`